src/bindings/python/src/pyopenvino/core/properties/properties.cpp

```diff
@@ -71,6 +71,12 @@ void regmodule_properties(py::module m) {
 
     // Submodule intel_cpu property
     wrap_property_RW(m_intel_cpu, ov::intel_cpu::denormals_optimization, "denormals_optimization");
+    wrap_property_RW(m_intel_cpu,
+                     ov::intel_cpu::experimental_fc_sparse_wei_decompression_enable,
+                     "experimental_fc_sparse_wei_decompression_enable");
+    wrap_property_RW(m_intel_cpu,
+                     ov::intel_cpu::experimental_fc_sparse_wei_decompression_rate,
```

Comment:
1. I would propose to simplify: ov::intel_cpu::sparse_weights_decompression_rate 
2. Since we haven't implemented good heuristic to allow automatic switch for sparse path, lets leave only rate key. 1.0f - default value meaning sparsity is disabled.

Reply:
done